RELEASE/scripts/sl_ascend/sl_combat.ash

```diff
@@ -1674,6 +1674,7 @@ string sl_combatHandler(int round, string opp, string text)
 			stunner = useSkill($skill[Soul Bubble]);
 			costStunner = mp_cost($skill[Soul Bubble]);
 		}
+		loopHandlerDelayAll();
```

Comment:
What is this one for? Unless I'm reading it wrong it just applies to every round of combat as a sauceror.